src/main/java/view/CarOutputView.java

```diff
@@ -0,0 +1,34 @@
+package view;
+
+import model.RacingGame;
+
+import java.util.List;
+
+public class CarOutputView {
+    public void printCarNameMessage() {
+        System.out.println("경주할 자동차 이름을 입력하세요(이름은 쉼표(,)를 기준으로 구분).");
+    }
+
+    public void printTryNumberMessage() {
+        System.out.println("시도할 회수는 몇 회인가요?");
+    }
+
+    public void printResultMessage() {
+        System.out.println("실행 결과");
+    }
+
+    public void printCarResult(List<String> carMessages) {
+        carMessages.forEach(System.out::println);
+        System.out.println();
+    }
+
+    public void printWinnerMessage(RacingGame racingGame) {
+        StringBuilder sb = new StringBuilder();
+        racingGame.getWinners().forEach(car -> sb.append(car.getName()).append(", "));
```

Comment:
```suggestion
        String result = racingGame.getWinners().stream().map(Car::getName).collect(Collectors.joining(", "));
        System.out.println(result + "가 최종 우승했습니다.");
```
이렇게 하면 -2 라는 매직 넘버를 사용하지 않고도 구현이 가능할 것 같아요!

Reply:
자바 언어가 익숙하지 않아서 고민했는데 감사합니다!! [7bea4d4](https://github.com/next-step/java-racingcar-kakao/pull/4/commits/7bea4d40b7208dd2c9bb88e72ec59b5cde567687)